internal/build/phase_config_provider.go

```diff
@@ -47,6 +47,8 @@ func NewPhaseConfigProvider(name string, lifecycleExec *LifecycleExecution, ops
 	provider.ctrConf.Labels = map[string]string{"author": "pack"}
 
 	if lifecycleExec.opts.MacAddress != "" {
+		// TODO fix this
+		//nolint:staticcheck
 		provider.ctrConf.MacAddress = lifecycleExec.opts.MacAddress
```

Comment:
Is there are __real__ test for MAC? There seems to be two superficial tests that do not actually test that MAC is set for actual container.

Reply:
I am considering this:
https://github.com/buildpacks/pack/pull/2063/commits/3130aeed5a96eba5a3342c02999a35a5be6838d4
but I don't know if it would work.

Reply:
I think for now we can live with the deprecation warning. Right?

Reply:
Yeah, also the mac-address feature has not being released yet, I think I can take a look later